### PR TITLE
add function backupClipboard, function restoreClipboard

### PR DESCRIPTION
**Pull request by xingwozhonghua126**

### CODE REVIEW

**Comment by xingwozhonghua** [comment]
If I didn’t write it correctly, please forgive me. I don’t understand Win32 programming. I’ll let ChatGPT complete it.

File: QuickLook.Native/QuickLook.Native32/Everything.cpp [hunk 1]
```diff
@@ -52,10 +53,44 @@ void Everything::GetSelected(PWCHAR buffer)
...
+        CloseClipboard();
+		return;
+    }
+
+    backupData = GlobalAlloc(GMEM_MOVEABLE, sizeof(WCHAR) * (wcslen(static_cast<PWCHAR>(GlobalLock(hData))) + 1));
+    if (backupData != nullptr)
+	{
+        memcpy(GlobalLock(backupData), GlobalLock(hData), sizeof(WCHAR) * GlobalSize(hData));
+        GlobalUnlock(backupData);
+    }
```
Comment (xupefei) [inline QuickLook.Native/QuickLook.Native32/Everything.cpp:71]: `Operations on `backupData` are not atomic. There should be a lock to protect this range and the read path below.

**Comment by xingwozhonghua** [comment]
@xupefei Could you please take a look and see if this change is okay?

**Comment by xingwozhonghua** [comment]
https://github.com/QL-Win/QuickLook/issues/1322

**Pull request closed without merging**